pkgs/development/compilers/coreclr/default.nix

```diff
@@ -41,67 +41,43 @@ stdenv.mkDerivation rec {
     lttng-ust
     liburcu
     libuuid
-    ed
+    libkrb5
   ];
 
   configurePhase = ''
-    # Prevent clang-3.5 (rather than just clang) from being selected as the compiler as that's
-    # not wrapped
-    # substituteInPlace src/pal/tools/gen-buildsys-clang.sh --replace "which \"clang-\$" "which \"clang-DoNotFindThisOne\$"
-
-    patchShebangs build.sh
-    patchShebangs src/pal/tools/gen-buildsys-clang.sh
-
-    # See https://github.com/dotnet/coreclr/issues/7573#issuecomment-253081323
-    ed -v ./src/pal/src/include/pal/palinternal.h << EOF
-    /^#undef memcpy
-    -1
-    d
-    +1
-    d
-    w
-    EOF
+    # override to avoid cmake running
+    patchShebangs .
   '';
 
   BuildArch = if stdenv.is64bit then "x64" else "x86";
   BuildType = if debug then "Debug" else "Release";
 
-  hardeningDisable = [ "strictoverflow" "format" ];
-  NIX_CFLAGS_COMPILE = [
-    "-Wno-error=unused-result" "-Wno-error=delete-non-virtual-dtor"
-    "-Wno-error=null-dereference"
+  hardeningDisable = [
+    "strictoverflow"
+    "format"
   ];
 
   buildPhase = ''
-    ./build.sh $BuildArch $BuildType
-
-    # Try to make some sensible hierarchy out of the output
-    pushd bin/Product/Linux.$BuildArch.$BuildType
-    mkdir lib2
-    mv *.so *.so.dbg lib2
-    mv bin lib3
-    mkdir lib4
-    mv Loader lib4
-    mv inc include
-    mv gcinfo include
-    mkdir bin
-    mkdir -p share/doc
-    mv sosdocsunix.txt share/doc
-    for f in * ; do test -f $f && mv -v $f bin; done
-    popd
+    runHook preBuild
+    ./build.sh $BuildArch Release
+    runHook postBuild
   '';
 
   installPhase = ''
-    mkdir -p $out
-    cp -rv bin/Product/Linux.$BuildArch.$BuildType/* $out
+    runHook preInstall
+    mkdir -p $out/share/dotnet $out/bin
+    cp -r bin/Product/Linux.$BuildArch.$BuildType/* $out/share/dotnet
+    for cmd in coreconsole corerun createdump crossgen ilasm ildasm mcs superpmi; do
+      ln -s $out/share/dotnet/$cmd $out/bin/$cmd
+    done
+    runHook postInstall
   '';
 
-  meta = {
+  meta = with stdenv.lib; {
     homepage = http://dotnet.github.io/core/;
     description = ".NET is a general purpose development platform";
     platforms = [ "x86_64-linux" ];
-    maintainers = with stdenv.lib.maintainers; [ obadz ];
-    license = stdenv.lib.licenses.mit;
-    broken = true; # CoreCLR has proven to be very difficult to package. PRs welcome if someone wants to shave that yak.
+    maintainers = with maintainers; [ obadz ];
```

Comment:
@obadz do you still want to be a maintainer here? and @kuznero are you interested in being one?

Reply:
@copumpkin yes, I am interested

Reply:
@copumpkin: no, feel free to remove me. Cheers.

Reply:
@kuznero maybe just add yourself to this list in this PR and remove @obadz while you're at it? This isn't my PR so I can't remove you myself @obadz